src/InputOutput/Writers/netcdf.jl

```diff
@@ -0,0 +1,32 @@
+using NCDatasets
+
+"""
+    write_data(filename::String,
+               dimnames::Tuple,
+               dims::Tuple,
+               axes::Tuple,
+               varnames::Tuple,
+               varvals::Array)
+
+Writes dimensions, axes and variables to a NetCDF file. Supports any number
+of dimensions.
+"""
+function write_data(filename, dimnames, dims, axes, varnames, varvals)
+    nd = length(dimnames)
+    nv = length(varnames)
+    clns = [Colon() for _ in 1:nd]
+    Dataset(filename, "c") do ds
+        for n in 1:nd
+            ds.dim[dimnames[n]] = dims[n]
+        end
+        for n in 1:nd
+            defVar(ds, dimnames[n], axes[n], (dimnames[n],))
+        end
+        for v in 1:nv
+            @eval (defVar($ds, $(varnames[v]), $(varvals[clns...,v]),
+                          $(dimnames)))
```

Comment:
does this work without the `@eval`?

Reply:
🤔 ... I don't see why it wouldn't. I just copied @skandalaCLIMA's code over without thinking about it. Why did you use `@eval` @skandalaCLIMA?

Reply:
I was using it to save the handle for the variable. Comments and other attirbutes can be added based on the handle. However, it is not currently used and I agree that it can be removed.